docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,216 @@
+4. Your new Code Builder environment is now available on your dashboard.
+5. Click **Launch** to launch Code Builder in a new tab. 
+6. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a minute (well, currently  x-y minutes) to start up when it initially sets up.
+
+**Tip**: Once you have created your Code Builder environment, just Launch your Code Builder environment from the dashboard at any time, or bookmark it for fast access.
+
+## Let's Take A Quick Tour
+You’re now in your developer environment in Code Builder. Code Builder has VS Code’s rich IDE plus it gives you easy access to cool Salesforce development-specific tools through the Salesforce extension pack. Let’s take a quick tour and get to know the lay of the land. Your screen looks something like this:
+
+![code_builder_ui](../../../images/code_builder_ui.jpg)
+The Code Builder user interface is divided into five main areas.
+
+**Activity Bar**: Located on the left-hand side and it contains iconic buttons to switch between different views. For instance, the Explorer view is active in the above picture.
+**Side Bar**: Contains different views like the Explorer to assist you while working on your project. For example, you can see the Explorer view active in the above image which lists the files in a file structure of your project.
+**Editors**: The main areas to edit your files. You can open as many editors as you like side by side, vertically and horizontally.
+**Panel**: Shows output or debug information, errors and warnings, or an integrated terminal. You can choose the output you want to display.
+**Status Bar**: Shows information about the opened project and the files you’re editing. This is where you can also see the org you’re working against. 
+
+## Get To Know Your Org – Use the Org Browser
+Take a closer look at the activity bar and notice a cloud icon. This is the Org Browser. It’s a part of the Salesforce Extension Pack. It helps you browse and retrieve metadata from your org without having to use a manifest file. The Org Browser is available only in non source-tracked orgs such as sandboxes or developer edition orgs. You can retrieve the metadata source into your local project. 
+
+**More Information**: You can find more information about the [Org Browser](https://developer.salesforce.com/tools/vscode/en/user-guide/org-browser) in the Salesforce Extension Pack documentation.
+
+![org_browser_small](../../../images/org_browser_small.jpg)
+
+
+Click Org Browser and scroll down to **Custom Objects**. The first object you see is Account. You can view the metadata of the **Account** object (and do a few more cool things, but we’ll get to those in a bit). 
+
+## Use SOQL Builder to Build and Run your First Query
+
+Code Builder comes preloaded with everything you need, to build and run a SOQL query as a part of the Salesforce extension pack. 
```

Comment:
Here SF extension pack is l/c. I don't believe you want a comma between need and to.